cdap-cli-tests/src/test/java/io/cdap/cdap/cli/CLITestBase.java

```diff
@@ -272,12 +273,13 @@ public void testPrompt() throws Exception {
   @Test
   public void testProgram() throws Exception {
     ProgramClient programClient = getProgramClient();
-    final ProgramId serviceId = FAKE_APP_ID.service(FakeApp.SERVICES.get(0));
+    final ProgramId serviceId = FAKE_APP_ID_V_1.service(FakeApp.SERVICES.get(0));
```

Comment:
just nit : it is better to remove the versioned app `FAKE_APP_ID_V_1` instead of the original `FAKE_APP_ID` for readability?

Reply:
sry I don't understand, can you explain more on this?